design/multiple-storage-types-support.md

```diff
@@ -340,6 +370,13 @@ A source code layout that includes these new additions is shown below, annotated
       - minio
 ```
 
+## Summary
+
+* Rook will enable storage providers to integrate their solutions with cloud-native environments by providing a framework of common abstractions and implementations that helps providers efficiently build reliable and well tested storage controllers.
+  * `StorageScopeSpec`, `PlacementSpec`, `ResourceSpec`, `NetworkSpec`, etc.
+* Each storage provider will be versioned independently with its own API Group (e.g., `ceph.rook.io`) and its own instantiable CRD type(s).
+* Each storage provider will have its own operator pod that performs orchestration and management of the storage resources.  This operator will use a provider specific container image with any special tools needed.
```

Comment:
since each backend has its own operator, the `version` in the cluster crd would not be necessary, as mentioned in the previous comment, right?

Reply:
Yes, i think you are correct.  I think I got the operator manifest and the backend specific cluster CRD muddled in my head a bit with respect to container image version.